BuildRelease.msbuild

```diff
@@ -107,7 +107,7 @@
         <ItemGroup>
           <XunitTestAssemblies
               Include="Src\*Test\bin\$(Configuration)\*Test.dll"
-              Exclude="Src\AutoFixture.NUnit2.*Test\bin\$(Configuration)\*.dll" />
+              Exclude="Src\AutoFixture.NUnit*.*Test\bin\$(Configuration)\*.dll" />

```

Comment:
This prevents the single test under AutoFixture.NUnit3.UnitTest from running. Without this change the build would fail as xUnit runners will include it in assemblies.